sources/packages/backend/libs/sims-db/src/entities/student-file.model.ts

```diff
@@ -95,4 +96,24 @@ export class StudentFile extends RecordDataModel {
     nullable: true,
   })
   metadata?: StudentFileMetadata;
+  /**
+   * Virus scan status of the file.
+   */
+  @Column({
+    name: "virus_scan_status",
+    nullable: false,
+    type: "enum",
+    enum: VirusScanStatus,
+    enumName: "VirusScanStatus",
+  })
+  virusScanStatus: VirusScanStatus;
+  /**
+   * Date and time when the virus scan status of the file was updated.
+   */
+  @Column({
+    name: "virus_scan_status_updated_on",
+    type: "timestamptz",
+    nullable: false,
```

Comment:
nullable should be true right? for initial row update?

Reply:
There is a default value to this column.